### DISCUSSION TITLE
Registration on Anygenre 3 - client problem

### DISCUSSION THREAD

**Discussion by 68rich** [jamulussoftware/jamulus#3170] [Technical Support & Help]
Hi,
I am running a server registered on Anygenre3. THis all seems fine except that one client in particular can;t see the server in the connection window in the jamulus client. All the other servers in that directory seem to be visible.
What I am finding particularly strange is that if she browses to explore.jamulus.io, the server is visible.
I've tried a few things - restarting the server - registering on another directory etc. but still no dice.
Does anyone have any suggestions about where to look?

Cheers,

Richard.

**Comment by dcorson-ticino-com** [comment]
Does it work if you give her the IP address of the server which she can enter in the "Server Address" field of the connect window ?
It may be that for some strange reason she has no access through her provider to your server.  There is then not much one can do except to change the provider.  
Note that the info for explore.jamulus.io goes through other servers from you to her so it doesn't tell us anything about the direct connection between your server and her client.
Another possibility to debug would be to use her cellphone as a hotspot and connect through that.  It will be way too slow to give good performance with Jamulus, but as it probably uses another provider it could give info if that is really the problem.

**Comment by mcfnord** [comment]
When a server does not appear on the Connection list on a client, it's probably because that client can't reach that server. This means entering the IP:PORT manually is unlikely to help. Other clients might reach the server, including the Jamulus Explorer data collector.

Reasons why this happen can include well-meaning security settings at one of the two ISPs involved. These can be hard to diagnose and change.

If a cloud server can work for this group, this is a pretty good argument for using one.
